node-tests/acceptance/exam-test.js

```diff
@@ -13,6 +13,12 @@ function getNumberOfTests(str) {
 
 var TOTAL_NUM_TESTS = 55;
 
+function getTotalNumberOfTests(str) {
+  // Ember.onerror validation test is added per browser, so the total number of tests executed is sum of 55 and a number of browsers.
```

Comment:
Lets also point out how this is added:
In ember-qunit 3.4.0, this new check was added: [https://github.com/emberjs/ember-qunit/commit/a7e93c4b4b535dae62fed992b46c00b62bfc83f4](url)
which adds this `Ember.onerror validation test`.